Add information about `only_logs_after` and `reparse` in AWS integration's documentation (#4650)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/wodle-s3.rst

bucket\\only_logs_after
^^^^^^^^^^^^^^^^^^^^^^^

A valid date, in YYYY-MMM-DD format. Only logs from that date onwards will be parsed.

+--------------------+-------------+

> | **Default value**  | None        |

Reviewer: This default value is wrong

.. versionadded:: 4.0.0

> A valid date, in YYYY-MMM-DD format. Only logs from that date onwards will be parsed. Only works with the CloudWatch Logs service.

Reviewer: ```suggestion
A valid date, in YYYY-MMM-DD format. Only logs from that date onwards will be parsed. This option is only available for the CloudWatch Logs service.
```

+--------------------+-------------+

> | **Default value**  | None        |

Reviewer: Same here
